RELEASE-NOTES.txt

```diff
@@ -3,6 +3,8 @@
 
 22.0
 -----
+- [*] Payments: Puerto Rico is now available in the list of countries that are supported by In-Person Payments, when using WooPayments [https://github.com/woocommerce/woocommerce-ios/pull/14972]
```

Comment:
For some reason, we have added two very similar release notes. Maybe the result of some conflict resolution? Let's remove one of them 👍

Reply:
Good catch 🤦 , yes, it seems to come from a conflict between the latest 2 trunk merges, as in one of them we updated the note and didn't catch it as a conflict. Fixed: 6430c384bea4e13e751880bf6f6b6b5b3007811f